scm-ui/ui-components/src/Icon.tsx

```diff
@@ -39,12 +40,12 @@ export default class Icon extends React.Component<Props> {
   };
 
   render() {
-    const { title, iconStyle, name, color, className } = this.props;
+    const { title, iconStyle, name, color, className, onClick } = this.props;
     if (title) {
       return (
-        <i title={title} className={classNames(iconStyle, "fa-fw", "fa-" + name, `has-text-${color}`, className)} />
+        <i onClick={onClick} title={title} className={classNames(iconStyle, "fa-fw", "fa-" + name, `has-text-${color}`, className)} />
       );
     }
-    return <i className={classNames(iconStyle, "fa-" + name, `has-text-${color}`, className)} />;
+    return <i onClick={onClick} className={classNames(iconStyle, "fa-" + name, `has-text-${color}`, className)} />;
   }
 }
```

Comment:
Why do we need an onClick on an Icon? I think it is better to create an element around the icon, instead of adding an onClick handler to the icon.

By the way, the onClick is not used in this pr.

Reply:
Removed it.